### PR TITLE
Remove support of Ubuntu16 and CentOS8 epe940

### PR DESCRIPTION
**Pull request by softprofe**
## Change Description

Remove support of Ubuntu 16 and CentOS 8, see  also EPE 940

## Change Type
**Select *ONE*:**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [x] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x] Documentation Additions

### CODE REVIEW

File: scripts/eosio_build_centos.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ echo "Disk space total: ${DISK_TOTAL}G"
 echo "Disk space available: ${DISK_AVAIL}G"
 
 ( [[ $NAME == "CentOS Linux" ]] && [[ "$(echo ${VERSION} | sed 's/ .*//g')" < 7 ]] ) && echo " - You must be running Centos 7 or higher to install EOSIO." && exit 1
-
+( [[ $NAME == "CentOS Linux" ]] && [[ "$(echo ${VERSION} | sed 's/ .*//g')" == 8 ]] ) && echo " - We don't support CentOS 8 any more ." && exit 1
```
Comment (kj4ezj) [inline scripts/eosio_build_centos.sh:9]: >  \- We don't support CentOS 8 any more .  
  
Should we make this message more specific or helpful? What about this?

> CentOS 8 has been deprecated by the RHEL community and is no longer supported by EOSIO. Please use CentOS 7 or Amazon Linux 2 instead.

File: scripts/generate_deb.sh [hunk 2]
```diff
@@ -26,8 +26,6 @@ fi
 DISTRIB_MAJOR_VERSION="$(echo "$DISTRIB_RELEASE" | cut -d '.' -f '1')"
 if (( "$DISTRIB_MAJOR_VERSION" >= 18 )); then
     RELEASE_SPECIFIC_DEPS='libssl1.1'
-elif (( "$DISTRIB_MAJOR_VERSION" >= 16 )); then
-    RELEASE_SPECIFIC_DEPS='libssl1.0.0'
 else
     echo "Found Ubuntu $DISTRIB_MAJOR_VERSION, but not sure which version of libssl to use. Exiting..."
     exit 2
```
Comment (kj4ezj) [inline scripts/generate_deb.sh:31]: Is Ubuntu 20.04 missing here?

**Review by kj4ezj — CHANGES_REQUESTED** [review]
Couple things, nothing too serious.

File: scripts/generate_deb.sh (hunk 2 above)
Comment (softprofe) [reply scripts/generate_deb.sh:31]: yes, seems no 20 so just change to 20.

File: scripts/eosio_build_centos.sh (hunk 1 above)
Comment (softprofe) [reply scripts/eosio_build_centos.sh:9]: Good sentence.

**Review by kj4ezj — APPROVED** [review]
🍕

**Pull request merged**